internal/plans/changes.go

```diff
@@ -560,9 +560,15 @@ func (i *Importing) Encode() *ImportingSrc {
 	if i == nil {
 		return nil
 	}
-	if i.ID.IsKnown() {
-		return &ImportingSrc{
-			ID: i.ID.AsString(),
+	if i.Target.IsKnown() {
```

Comment:
This needs to be `IsWhollyKnown`, because it would be the nested attributes which are not known.